lib/ansible/plugins/inventory/linode.py

```diff
@@ -76,27 +77,14 @@ def _build_client(self):
         """Build the Linode client."""
 
         access_token = self.get_option('access_token')
-
-        if access_token is None:
```

Comment:
This conditional was removed but not moved somewhere else?

Reply:
This should be handled in the documentation, which doubles as an argument specification for inventory plugins. The access_token is listed as required, and it falls back to the LINODE_ACCESS_TOKEN environment variable. If the option isn't provided, the plugin should fail.